packages/tokamak/contracts-bedrock/hardhat.config.ts

```diff
@@ -0,0 +1,60 @@
+import "@nomiclabs/hardhat-waffle";
+// import "@nomicfoundation/hardhat-verify";
+import "@nomiclabs/hardhat-etherscan";
+import 'hardhat-deploy';
+
+const LOWEST_OPTIMIZER_COMPILER_SETTINGS = {
+  version: '0.7.6',
+  settings: {
+    evmVersion: 'istanbul',
+    optimizer: {
+      enabled: true,
+      runs: 1_000_000,
+    },
+    metadata: {
+      bytecodeHash: 'none',
+    },
+  },
+};
+
+export default {
+  solidity: {
+    compilers: [
+      LOWEST_OPTIMIZER_COMPILER_SETTINGS,
+    ],
+  },
+  networks: {
+    devnetL1: {
+      url: "http://localhost:9545",
+      chainId: 901,
+    },
+    'thanos-sepolia-test': {
+      url: "https://explorer.thanos-sepolia-test.tokamak.network",
```

Comment:
-> https://rpc.thanos-sepolia-test.tokamak.network 🙇🏼‍♂️😍

Reply:
Thank you for your correction!!